packages/hardhat/contracts/FundingVault.sol

```diff
@@ -56,13 +57,34 @@ contract FundingVault{
     uint256 private blockLimit;
     uint256 private minFundingAmount; //The minimum amount of ETH required in the contract to enable withdrawal.
     uint256 private exchangeRate; //The exchange rate of ERG per token
+    address private projectOwner;
     address private withdrawlAddress;
     address private developerFeeAddress; //develper address
     uint256 private developerFeePercentage; 
     string  private projectURL;
+    string private projectTitle;
+    string private projectDescription;
     bool private fundsWithdrawn;
 
 
+    /** 
+    * @dev A vault is represented as a struct  
+    */ 
+
+    struct Vault{
+        address withdrawlAddress;
+        address participationToken;
+        uint256 participationTokenAmount;  
+        uint256 minFundingAmount;
+        uint256 blockLimit;
+        uint256 exchangeRate;
+        string projectURL;
+        string projectTitle;
+        string projectDescription;
+    }
+    
+
+
     // Events //
     event TokensPurchased(address indexed from, uint256 indexed amount);
     event Refund(address indexed user, uint256 indexed amount);
```

Comment:
It is better to remove the modifiers `deadlinePassed` and `deadlineNotPassed` and implement their logic directly in the respective function.

Also, the "<" and ">" are swapped in the two conditionals.

Reply:
Fixed: Modifiers are removed in latest commit.